multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -0,0 +1,291 @@
+import anyTest from '@endo/ses-ava/prepare-endo.js';
+import type { TestFn } from 'ava';
+import { AmountMath } from '@agoric/ertp';
+import type { Denom } from '@agoric/orchestration';
+import { divideBy } from '@agoric/zoe/src/contractSupport/ratio.js';
+import type { IBCChannelID } from '@agoric/vats';
+import { makeDoOffer, type WalletDriver } from '../../tools/e2e-tools.js';
+import { makeDenomTools } from '../../tools/asset-info.js';
```

Comment:
`makeDenomTools` looks pretty interesting. We should keep an eye out for re-invention of similar things and migrate them to the orch package.

Reply:
Good feedback. I'll see if this worth moving to `@agoric/orchestration` in a future PR. It is somewhat like `assetOn` - maybe that should be moved to `@agoric/orchestration` tools as well.

Reply:
> It is somewhat like `assetOn`

In fact, I sorta I assumed it was a wrapper around `assetOn` without reading it carefully.